Documentation/StyleGuide.md

```diff
@@ -75,6 +75,35 @@ documentation. Symbols that fulfill protocol requirements do not need to be
 given additional documentation (the documentation in the protocol declaration is
 generally sufficient.)
 
+
+When writing an abstract for a symbol, start the abstract with either a noun or
+a verb ending in s depending on what kind of symbol it is:
+|   Noun               |   Verb ending in *s*      |
+|----------------------|---------------------------|
+| Associated type      | Enumerations |
+| Class                | Function and function macro |
+| Constant             | Initializer |
+| Enumerated types     | Macro |
+| Property             | Method |
+| Protocol             | Subscript |
+| Structure            |  |
+| Type alias           |  |
+| Variable             |  |
+
+For instance, when writing the abstract for a class `Order`, you could write:
+<blockquote>
```

Comment:
Don't use `<blockquote>` or other HTML here. Instead, you can use `> ` to indicate that a line is block-quoted:

```markdown
> An object that stores the details for a specific order from a vendor.
```